<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>504</width>
    <height>510</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>504</width>
    <height>510</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>504</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="openFileButton">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>120</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Click to open a file</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelFile">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../icons/resources.qrc">:/folder_open/folder_open.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="indent">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="removeFileButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../icons/resources.qrc">
      <normaloff>:/delete_file/delete_file.png</normaloff>:/delete_file/delete_file.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>25</width>
      <height>25</height>
     </size>
    </property>
   </widget>
   <widget class="QTextEdit" name="textBoxMD5">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>331</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="lineWrapColumnOrWidth">
     <number>0</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelMD5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>MD5</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textBoxSHA256">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>331</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="lineWrapColumnOrWidth">
     <number>0</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="tabStopWidth">
     <number>80</number>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSHA256">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>220</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA256</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textBoxSHA512">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>270</y>
      <width>331</width>
      <height>91</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="lineWrapMode">
     <enum>QTextEdit::WidgetWidth</enum>
    </property>
    <property name="lineWrapColumnOrWidth">
     <number>0</number>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="tabStopWidth">
     <number>80</number>
    </property>
    <property name="placeholderText">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelSHA512">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>270</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>SHA512</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDoneMD5">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="indent">
     <number>-1</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelDoneSHA256">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="labelDoneSHA512">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>381</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../icons/resources.qrc">:/header/header.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="textBoxCheck">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>331</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>450</y>
      <width>88</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clipboardButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>450</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Paste clipboard</string>
    </property>
   </widget>
   <widget class="QPushButton" name="checkButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>450</y>
      <width>91</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Check</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelHashBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>380</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;HashBox&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clipboardMD5Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../icons/resources.qrc">
      <normaloff>:/clipboard/clipboard.png</normaloff>:/clipboard/clipboard.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>27</width>
      <height>31</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clipboardSHA256Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../icons/resources.qrc">
      <normaloff>:/clipboard/clipboard.png</normaloff>:/clipboard/clipboard.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>27</width>
      <height>31</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="clipboardSHA512Button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="../icons/resources.qrc">
      <normaloff>:/clipboard/clipboard.png</normaloff>:/clipboard/clipboard.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>27</width>
      <height>31</height>
     </size>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>504</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../icons/resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>textBoxCheck</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>160</x>
     <y>486</y>
    </hint>
    <hint type="destinationlabel">
     <x>164</x>
     <y>442</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
